config/locales/ar.yml

```diff
@@ -1051,8 +1051,6 @@ ar:
               رسالة اختيارية للترحيب بالعملاء وشرح كيفية التسوق معك. إذا تم إدخال
               النص هنا ، فسيتم عرضه في علامة تبويب الصفحة الرئيسية عندما يصل العملاء
               إلى واجهة المتجر الخاصة بك.
-          shopfront_message_link_tooltip: "إدراج / تحرير الرابط"
-          shopfront_message_link_prompt: "الرجاء إدخال عنوان URL لإدراجه"
```

Comment:
Please remove all changes to locales other than en.yml.

Reply:
:+1: Fixed by https://github.com/openfoodfoundation/openfoodnetwork/pull/12734/commits/dd5175558e7123052e900ea00b93d29df8a2cc4f